osfoffline/settings/defaults.py

```diff
@@ -16,7 +16,7 @@
 POLL_DELAY = 5 * 60  # seconds
 
 # Time to keep alert messages on screen (in milliseconds); may not be configurable on all platforms
-ALERT_TIME = 1000  # ms
+ALERT_TIME = 10000  # ms
```

Comment:
I'm guessing this is a remnant of local debugging efforts, rather than something intentionally committed?